t/web/download.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+require 'test_helper'
+require_relative '../../app'
+
+describe 'Viewer' do
+  subject { Nokogiri::HTML(last_response.body) }
+
+  describe 'when viewing the home page' do
+    before { get '/united-states-of-america/download.html' }
+
+    it 'should have show some text' do
+      last_response.body.must_include 'https://cdn.rawgit.com/everypolitician/everypolitician-data/'
```

Comment:
This will find this anywhere in the body, e.g. on the "Download CSV" links for the term data.